views/default/object/elements/summary.php

```diff
@@ -49,7 +49,9 @@
 if ($title_link) {
 	echo "<h3>$title_link</h3>";
 }
-echo "<div class=\"elgg-subtext\">$subtitle</div>";
+if ($subtitle) {
```

Comment:
User can use ` ` (one blank space) or `<!-- -->` to get an empty element if that's what you mean.